.gitignore

```diff
@@ -10,3 +10,6 @@ datalink/*.zip
 build
 python/lsst_sdm_schemas.egg-info
 __pycache__
+
+# LSST version
+python/lsst/sdm_schemas/version.py
```

Comment:
That was going to be needed for SCons builds as well. You might want to add `config.log` to the list as well, which scons writes.

Reply:
Will do.